src/modules/mainSec.js

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable import/no-cycle */
+import NewItem from './listClass.js';
+import * as selectors from './selectors.js';
+import { clear } from './interactive.js';
+
+export default class UpdateUi {
+  static taskArr = []
+
+  static enterBtn = selectors.enterBtn;
+
+  static input = selectors.input;
+
+  static listblock = selectors.listblock;
+
+  static clear = selectors.clear;
+
+  // populating the ul with task
+
+  static updateTask = () => {
+    const taskItem = new NewItem(UpdateUi.input.value);
+    UpdateUi.taskArr.push(taskItem);
+    UpdateUi.taskArr.forEach((taskItem, pos) => {
+      taskItem.index = pos + 1;
+    });
+    localStorage.setItem('taskItems', JSON.stringify(UpdateUi.taskArr));
+    UpdateUi.input.value = '';
+    UpdateUi.addTasks(UpdateUi.taskArr);
+  }
+
+  static delTask = (taskItem, index) => {
+    const taskBlock = document.getElementById(index);
+    UpdateUi.taskArr = UpdateUi.taskArr.filter((item) => item !== taskItem);
+    UpdateUi.taskArr.forEach((taskItem, pos) => {
+      taskItem.index = pos;
+    });
+    localStorage.setItem('taskItems', JSON.stringify(UpdateUi.taskArr));
+    UpdateUi.listblock.removeChild(taskBlock);
+  }
```

Comment:
- [ ] **Required:** Great job implementing the delete button. However, I have noticed that when an item is deleted using it the index is updated but begins from 0 rather than the required 1. kindly fix this behaviour.

![fix_index](https://user-images.githubusercontent.com/27709832/171909266-badcfe13-4100-4f51-9b25-e36edcf27742.gif)